Triangle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Cheks on the type of triangle.
+ */
+public class Triangle {
+  
+  /*
+   * An array of sides values.
+   */
+  double[] sides;
+
+  /*
+   * Creates the Triangle object using (@link SideReading#readSide(double[]).
+   */
+  Triangle() {
+    sides = new double[3];
+    SideReading getting = new SideReading();
```

Comment:
Single responsibility violation, class Triangle must not read sides in any way - pass sides as constructor arguments